test/verify/check-storage-resize

```diff
@@ -139,7 +139,7 @@ class TestStorageResize(storagelib.StorageCase):
         # and unmount the fs explicitly.  But then we also need to
         # check it explicitly.
         #
-        self.machine.execute(f"(! findmnt -S '{fs_dev}' || umount '{fs_dev}') && fsadm -y check '{fs_dev}' && fsadm -y resize '{fs_dev}' '{size}'")
+        self.machine.execute(f"(! findmnt -S '{fs_dev}' || umount '{fs_dev}') && fsadm -y check '{fs_dev}' && fsadm -y resize '{fs_dev}' '{size}' && udevadm trigger")
```

Comment:
Please don't use `&&` in m.execute(), but `;`. These run with `set -e` and DTRT. `&&` does not do what you think it does:
```
❱❱❱ sh -exc 'false && true; echo NOTME'
+ false
+ echo NOTME
NOTME
```

Reply:
Done